addons/hr/models/hr_employee.py

```diff
@@ -1345,6 +1345,8 @@ def _get_versions_with_contract_overlap_with_period(self, date_from, date_to):
             '|', ('contract_date_end', '>=', date_from), ('contract_date_end', '=', False),
         ])
 
+    def get_avatar_card_data(self, fields):
+        return self.read(fields)
```

Comment:
```suggestion
        return self.read(fields)

```

Maybe adding a small space. 😇